.editorconfig

```diff
@@ -6,7 +6,3 @@ indent_style = tab
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
-
-[*.json]
```

Comment:
This exception was made to follow npm's formatting for package.json files. Is there a particular reason why you removed it?

Reply:
It conflicts with Prettier’s format otherwise